src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -176,13 +176,15 @@ function useChatUtilsHook(): UseChatUtilsHook {
     navigation.goBack();
   };
   const handleTextSystem = (item): string => {
+    let message = item?.description || item?.message;
     if (
       item?.rawJson?.userIdFollower === profile?.myProfile?.user_id ||
       item?.rawJson?.message?.userIdFollower === profile?.myProfile?.user_id
     ) {
-      return item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
+      message = item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
     }
-    return item?.description || item?.message;
+    return message;
+    return message;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a duplicate line of code at line 186 and 187. Both lines are returning the same variable `message`. You should remove one of them to avoid redundancy.
```diff
    let message = item?.description || item?.message;
    if (
      item?.rawJson?.userIdFollower === profile?.myProfile?.user_id ||
      item?.rawJson?.message?.userIdFollower === profile?.myProfile?.user_id
    ) {
      message = item?.rawJson?.textOwnMessage || item?.rawJson?.message?.textOwnMessage;
    }
-   return message;
    return message;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->